get_hashsets.js

```diff
@@ -0,0 +1,47 @@
+"use strict";
+
+const AppConstants = require("./app-constants").init();
+
+const fs = require("fs");
+const path = require("path");
+const request = require("request");
+
+const pkg = require("./package.json");
+
+const HIBP_AUTH = 'Bearer ' + new Buffer(AppConstants.HIBP_API_TOKEN);
+
+
+function getBreachHashset(breach) {
+  if (breach.IsActive && breach.IsVerified && breach.DataClasses.includes("Email addresses")) {
+    console.log("Active, verified breach with email addresses: " + breach.Name);
+    const url = AppConstants.HIBP_API_ROOT + '/enterprisesubscriber/hashset/' + breach.Name;
+    console.log("url: " + url);
+
+    const headers = {
+      'User-Agent': `${pkg.name}@${pkg.version}`,
+      'Authorization': HIBP_AUTH
+    }
+    const hashsetRequestObject = {
+      url: url,
+      headers: headers
+    };
+
+    request(hashsetRequestObject).pipe(fs.createWriteStream('breach_hashsets' + path.sep + breach.Name + '.zip'));
```

Comment:
Not sure which is more readable (`path.sep` vs `path.join()`), so I'll let you two debate it:

```js
fs.createWriteStream('breach_hashsets' + path.sep + breach.Name + '.zip')
```

vs

```js
fs.createWriteStream(path.join('breach_hashsets', `${breach.Name}.zip`)
```

Reply:
path.join() hands down. More than readability, let's not assume that the join() operation is a simple string concatenation. join() provides platform abstraction for us and also normalizes the resulting path string - let's use it.